### PR TITLE
[Doc Improvement] [Add verbiage linking to admin options in overview]

### PR DESCRIPTION
**Pull request by mobajemu-msft**
Add verbiage linking to admin options in overview and address doc-bug #8901

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b2aafba3fc1eab66740d1bbbc97ae525c02827ce/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0658b57-9e34-46cb-9012-4b5e23ded28a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9663d20d58170f3bf715523c273726a2a908b75f/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f711a2b-2ce5-4328-90cf-0d18802fdd08) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/overview.md [hunk 1]
```diff
@@ -23,7 +23,12 @@ The Teams app platform continues to evolve and expand holistically into the Micr
...
 
-Enrollment to [Microsoft 365 Targeted Release](/microsoft-365/admin/manage/release-options-in-office-365) and [Microsoft 365 Apps update channel](/deployoffice/change-update-channels) requires admin opt-in for the entire organization or selected users. For more information, see [Manage access to Teams apps across Microsoft 365](/microsoftteams/manage-third-party-teams-apps). Update channels are device specific and apply only to installations of Microsoft 365 running on Windows.
+Enrollment to [Microsoft 365 Targeted Release](/microsoft-365/admin/manage/release-options-in-office-365) and [Microsoft 365 Apps update channel](/deployoffice/change-update-channels) requires admin opt-in for the entire organization or selected users.
+
+> [!NOTE]
+> For additional admin options and guidance on your extended Teams app, please see the documentation [*'Teams apps that work on Outlook and Microsoft 365'*](/microsoft-365/admin/manage/teams-apps-work-on-outlook-and-m365)
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/overview.md:29]: Per MS Style Guide, "Avoid saying 'please' except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation."

Also I'm thinking it'd be nice to be more explicit that the link takes you to Admin docs. How about something like:

For additional admin guidance and options for managing your extended Teams app, see [Teams apps that work on Outlook and Microsoft 365] in the Microsoft 365 Admin Center documentation.

File: msteams-platform/m365-apps/overview.md [hunk 2]
```diff
@@ -23,7 +23,12 @@ The Teams app platform continues to evolve and expand holistically into the Micr
...
+
+Update channels are device specific and apply only to installations of Microsoft 365 running on Windows.
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/overview.md:31]: This sentence should be with the sentences above on TR and M365 Apps update channels before the !NOTE

**Review by erikadoyle — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5510c824707f96d44952c4c966181b98f0acaf18/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/996ac024-d2b2-483a-9488-418f8fa4e510) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — APPROVED** [review]

File: msteams-platform/m365-apps/overview.md [hunk 3]
```diff
@@ -23,7 +23,10 @@ The Teams app platform continues to evolve and expand holistically into the Micr
...
 
-Enrollment to [Microsoft 365 Targeted Release](/microsoft-365/admin/manage/release-options-in-office-365) and [Microsoft 365 Apps update channel](/deployoffice/change-update-channels) requires admin opt-in for the entire organization or selected users. For more information, see [Manage access to Teams apps across Microsoft 365](/microsoftteams/manage-third-party-teams-apps). Update channels are device specific and apply only to installations of Microsoft 365 running on Windows.
+Enrollment to [Microsoft 365 Targeted Release](/microsoft-365/admin/manage/release-options-in-office-365) and [Microsoft 365 Apps update channel](/deployoffice/change-update-channels) requires admin opt-in for the entire organization or selected users. Update channels are device specific and apply only to installations of Microsoft 365 running on Windows.
+
+> [!NOTE]
+> For additional admin guidance and options for managing your extended Teams app, see [*'Teams apps that work on Outlook and Microsoft 365'*](/microsoft-365/admin/manage/teams-apps-work-on-outlook-and-m365) in the Microsoft 365 Admin Center documentation
```
Comment (v-ypalikila) [inline msteams-platform/m365-apps/overview.md:29]: > [!NOTE]
> For more information on admin guidance and options for managing your extended Teams app, see [Teams apps that work on Outlook and Microsoft 365](/microsoft-365/admin/manage/teams-apps-work-on-outlook-and-m365).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/47a2b2625a8ae7ef93e37e1f9138756231282793/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/df6d5a44-d72a-4b40-85b9-26e88ef4a1df) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/86ccb1d3bdf6e6e153f66cd27a1a6e5a611cee73/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf44ec3b-acaf-424e-b95e-d0dc7c29cce7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/84ab10070ca63e4c4342aed8aa033a57ccb52a3f/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/134d64f7-72c9-4e9d-80db-c811fd395896) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**